src/endpoints/endpoints.ts

```diff
@@ -1,13 +1,75 @@
-import express, { Express, Request, Response } from 'express';
-import { addAlice } from '../db/db';
+import { Express, Request, Response } from 'express';
+import { createUsers, getUserByID, getAllUsers,  updateUserByID } from '../db/db';
+import { deletUsersById } from '../db/db';
+
 
 export function endpoints(app: Express) {
+  app.post('/addUser', (req: Request, res: Response) => {
+    try {
+    
+      const data: { 
+        usersid: number;
+        usersname: string | null; 
+        usersemail: string | null; 
+        userspassword: string | null; 
+        userstotalscore: number | null; 
+        userslevel: number | null; 
+      } =  req.body;
+
+      createUsers(data);
+      res.status(201).end();
+    } catch (error) {
+      console.log('Error creating user:', error);
+      res.status(500).json({ error: 'An error occurred while creating user' });
+    }
+  });
+
+
+  app.get('/users', async (req: Request, res: Response) => {
+    try {
+      const users = await getAllUsers()
+      res.json(users);
+    } catch (error) {
+      console.log('Error getting all users:', error);
+      res.status(500).json({ error: 'An error occurred while getting all users' });
+    }
+  });
+
+  app.get('/users/:id', async (req: Request, res: Response) => {
+    try {
+      const number = +req.params.id;
+      const user = await getUserByID(number)
 
-  app.get('/', (req: Request, res: Response) => {
-    res.send('BACKROW');
-    addAlice();
-    console.log('add alice from endpoints');
+      res.send(user);
+    } catch (error) {
+      console.log('Error getting user by id:', error);
+      res.status(500).json({ error: 'An error occurred while getting user by id' });
+    }
   });
 
-  // add more endpoints here
+  app.delete('/users/delete/:id', (req: Request, res: Response) => {
```

Comment:
same here with the name
```suggestion
  app.delete('/users/:id', (req: Request, res: Response) => {
```